README.md

```diff
@@ -2,29 +2,32 @@
 Compos-able processing pipelines
 
 ## Introduction
-You can use Pipes to create _named_ pipelines of actions (functions) which are executed in sequence. The return value of each function (action) is passed as a parameter to the next function (action) in the sequence. The return value of a pipeline is the return value of the last action in the sequence. 
+You can use Pipes to create pipelines of actions (functions) which are executed in sequence. The return value of each function (action) is passed as a parameter to the next function (action) in the sequence. The return value of a pipeline is the return value of the last action in the sequence. 
 
-Multiple pipelines can also be executed in sequence with the return value of one pipeline passed as a parameter to the next one.  
+By specifying multiple named _stages_ you can also group pipelines of actions in e.g. 'before', 'action' and 'after' actions, or in any other naming scheme you like. 
+Stage-grouped actions are executed in sequence with the return value of one stage passed as a parameter to the next one.  
 
 ## Usage
 ```javascript
-// Creates a new pipe with three stages, before, action, and after
-// all the before hooks should run, then all of the action hooks, then all of the after hooks
+// Creates a new pipe with three stages, before, action, and after. These are executed in the order they 
+// appear in the array.
+// For a given Pipe (in this example the pipe named'default' specified during the myPipes.on method calls) 
+// all the before hooks run first, then all of the action hooks, then all of the after hooks
 var myPipes = new Pipes(['before', 'action', 'after']);

```

Comment:
I understand the motivation for naming it `Pipes` however I prefer the term `Pipe` because it's a singular instance (not a collection). I would be interested in suggestions for a more descriptive name though, all I can think of at the moment is `ActionRouter`.